packages/react-instantsearch/examples/react-router/package.json

```diff
@@ -9,7 +9,7 @@
     "qs": "^6.3.0",
     "react": "^15.5.4",
     "react-dom": "^15.5.4",
-    "react-instantsearch": "beta",
+    "react-instantsearch": "4.1.0-beta.3",
```

Comment:
I want examples to rely on the latest existing version possible (latest or beta). Depending of the timeline we'll want them to be either on latest or beta (for instance, now, to be able to run the next-app).